src/main/app/app.component.spec.ts

```diff
@@ -19,13 +19,4 @@ describe('AppComponent', () => {
     const app = fixture.componentInstance;
     expect(app.title).toEqual('angular-template-project');
   });
```

Comment:
Are we still using 'angular-template-project' as project title? It should be Idea Hunt.

Reply:
fixed